docs(dedibox): add information for hw raid (#1781)
scaleway/docs-content: dedibox/hardware/reference-content/server-comparison-table.mdx

These comparison tables display our previous Dedibox offers and their equivalent current offers, grouped by categories.

<Message type="important">

>   The information provided below are for informative purposes only and have no contractual value.

Reviewer: ```suggestion
  The information provided below is for informative purposes only and has no contractual value.
```
